types/react-dom/test/react-dom-tests.tsx

```diff
@@ -229,6 +229,7 @@ function createRoot() {
     const root = ReactDOMClient.createRoot(document.documentElement);
 
     root.render(<div>initial render</div>);
+    root.render(false);
```

Comment:
Another kind of node: https://codesandbox.io/s/initialchildrne-accepts-reactnode-exy9jm